libs/crypto/src/threshsign/ThresholdSignaturesTypes.cpp

```diff
@@ -45,9 +45,20 @@ Cryptosystem::Cryptosystem(const std::string& sysType,
 
 // Helper function to generateNewPseudorandomKeys.
 IThresholdFactory* Cryptosystem::createThresholdFactory() {
+#ifdef USE_MULTISIG_BLS
+  bool is_bls = type_ == MULTISIG_BLS_SCHEME || type_ == THRESHOLD_BLS_SCHEME;
+  if (is_bls) {
+    bool use_multisig = type_ == MULTISIG_BLS_SCHEME || (type_ == THRESHOLD_BLS_SCHEME && forceMultisig_);
```

Comment:
why did BLS reappear here?

Reply:
moving threshsign and removing BLS were in separate commits